frontend/uddhar-frontend/src/shared/components/DashboardNavbar.jsx

```
@@ -32,17 +32,36 @@ const DashboardNavbar = ({ children, heading }) => {
           <Moon className="w-6 h-6 text-black cursor-pointer border rounded-full" />
           <Bell className="w-6 h-6 text-black cursor-pointer" />
           <div className="relative">
-            <button onClick={()=>{setUserMenuOpen(prev => !prev)}} className="cursor-pointer">
+            <button
+              onClick={() => {
+                setUserMenuOpen((prev) => !prev);
+              }}
+              className="cursor-pointer"
+            >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)